### DISCUSSION TITLE
Error when plotting smps.plots.heatmap

### DISCUSSION THREAD

**Discussion by Ethan-Barber** [quant-aq/py-smps#51] [General]
When I copy the code for the sample "boston" example and plot the heatmap, I receive the error "Cannot cast ufunc 'greater_equal' input 0 from dtype('<m8[ns]') to dtype('<m8') with casting rule 'same_kind'". Any ideas how I can fix this?

**Comment by dhhagan** [comment]
Hi @Ethan-Barber - so sorry for the delay in response here. I am going to transfer this discussion to an issue. Can you run `pip list` and post the versions of the packages you're using?
